rotkehlchen/exchanges/okx.py

```diff
@@ -273,7 +273,7 @@ def query_balances(self, **kwargs: Any) -> ExchangeQueryBalances:
                 continue
 
             try:
-                amount = deserialize_asset_amount(currency_data['availBal'])
+                amount = deserialize_asset_amount(currency_data['availBal']) + deserialize_asset_amount(currency_data['frozenBal'])
```

Comment:
This will fail linting. Add a noqa at the end
```suggestion
                amount = deserialize_asset_amount(currency_data['availBal']) + deserialize_asset_amount(currency_data['frozenBal'])  # noqa: E501
```